torch/jit/__init__.py

```diff
@@ -1342,6 +1342,10 @@ def wrapper(*args):
     return wrapper
 
 
+def to_test_backend(module, extra_info):
```

Comment:
I'd prefer not to do this for every backend, so if anyone has a way to programmatically generate one of these methods for every backend, please let me know.

Reply:
The intension in the design is that users do in fact write their own functions, to make it clear that bugs in the implementation should be submitted to the implementors of the backend and to as JIT bugs.